bonita-migration-db-filler-6.4.2/src/main/java/org/bonitasoft/migration/DatabaseChecker6_4_2.java

```diff
@@ -15,11 +15,14 @@
 
 import org.junit.runner.JUnitCore;
 
-
 public class DatabaseChecker6_4_2 extends DatabaseChecker6_4_1 {
-    
+
     public static void main(final String[] args) throws Exception {
         JUnitCore.main(DatabaseChecker6_4_2.class.getName());
     }
 
+    @Override
+    public void checkDates_are_retrieved_Correctly() throws Exception {
+        //nothing to do
```

Comment:
this check inherits DatabaseChecker6_4_1 and not a SimpleDatabaseChecker6_4_1 that not exists. migration to 6.4.2 fails if not filled from 6.3.9